pandas/io/parsers.py

```diff
@@ -1603,8 +1603,10 @@ def TextParser(*args, **kwds):
     has_index_names: boolean, default False
         True if the cols defined in index_col have an index name and are
         not in the header
-    na_values : iterable, default None
-        Custom NA values
+    na_values : scalar, str, list-like, or dict, default None
+        Additional strings to recognize as NA/NaN. If dict passed, specific
+        per-column NA values. By default the following values are interpreted
+        as NaN: '""" + "', '".join(sorted(_NA_VALUES)) + """'.
```

Comment:
Can you remove the join here? Concatenating docstrings does not work like this (as I experienced when reviewing the other PR which added this). 

And I don't think it is worth it for TextParser (not a very public class) to separate the docstring like is done for `read_csv`)